dist/pinny.js

```diff
@@ -467,6 +470,26 @@
             return $(document.activeElement);
         },
 
+        _isWithin: function($element, selector) {
+            return ($element.closest('.' + selector).length) ? true : false;
```

Comment:
You could rewrite the above like this:

```
return !!$element.closest('.' + selector).length;
```

But I think more importantly this would be a better match as a Zepto/jQuery extension. How about:

```
$.extend($.fn, {
    isChildOf: function(selector) {
        return $(this).parents().filter(selector).length > 0;
    }
});
```